File: backend/routes/metricas.py
from fastapi import APIRouter
from services import metricas as metricas_services

router = APIRouter(prefix="/metricas", tags=["metricas"])


@router.post("/")
def post_metricas():
    return metricas_services.criar_conversa()


@router.post("/feedback")
def post_feedback(ajudou: bool):
    return metricas_services.registrar_feedback(ajudou)


@router.get("/")
def get_metricas():
    return metricas_services.obter_metricas()

File: frontend/src/services/metricas.js
import {api} from './api';

export const postMetricas = async () => {
    const response = await api.post('/metricas/');
    return response.data;
}

export const getMetricas = async () => {
    const response = await api.get('/metricas/');
    return response.data;
}

export const postFeedback = async (isGood) => {
    return api.post(`/metricas/feedback?ajudou=${isGood}`);
};